articles/guides/migration-oauthro-oauthtoken.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Migrating From /oauth/ro to /oauth/token
+description: Learn how to migrate your API calls and responses from /oauth/ro to /oauth/token
+toc: true
+---
+# Migration Guide for Resource Owner Password Credentials Exchange
+
+The Resource Owner Password Credentials exchange is used by highly-trusted clients to provide active authentication. Unlike the authorization code and implicit grants, this authentication mechanism does not redirect users to Auth0. It authenticates users with a single request, exchanging their password credentials for a token. This document describes the differences of this flow between the legacy and OIDC-conformant authentication pipelines.
+
+Note that this guide is for users who use the resource owner password credentials exchange, and call /oauth/ro directly, without the use of any Auth0 libraries or SDKs. The major Auth0 libraries such as [Lock](/libraries/lock) or [Auth0.js](/libraries/auth0js) have already been updated to stop using /oauth/ro. If you use the `lock-passwordless` library, you can now use [Passwordless Mode](/libraries/lock/v11#passwordless) in Lock v11 instead.
+
+## Alter Your Requests
+
+Previously, requests to /oauth/ro looked similar to this:
+
+```json
+{
+  "grant_type": "password",
+  "client_id": "123",
+  "username": "alice",
+  "password": "A3ddj3w",
+  "connection": "my-database-connection",
+  "scope": "openid email favorite_color offline_access",
+  "device": "my-device-name"
+}
+```
+
+### Changes to Requests
+
+* The endpoint to execute token exchanges is now /oauth/token
+* [Auth0's own grant type](/api-auth/tutorials/password-grant#realm-support) is used to authenticate users from a specific connection `realm`. The [standard OIDC password grant](/api-auth/tutorials/password-grant) is also supported, but it does not accept Auth0-specific parameters such as `realm`.
+* `favorite_color` is no longer a valid scope.
```

Comment:
I think what we need to say here instead is something like (please wordsmith): "we support the standard OIDC scopes, along with scopes you have defined in your custom API".